packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -295,7 +303,16 @@ export default class ImageEdit implements EditorPlugin {
             this.image = image;
 
             // Get initial edit info
-            this.editInfo = getEditInfoFromImage(image);
+            if (this.editInfo) {
```

Comment:
Isn't this always false?  We set editInfo to null in line 296.

You need to check if any of the following exists in the image when getEditInfoFromImage, and maybe still make wasResized as part of ImageEditInfo (sorry for the back and force idea change, but you can also argue if you also fount it makes more sense to put wasResized into ImageEditInfo):
- img.style.width
- img.style.height
- img.width
- img.height